Neos.ContentRepository.Core/Classes/SharedModel/Node/NodeIdentity.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\SharedModel\Node;
+
+use Neos\ContentRepository\Core\DimensionSpace\DimensionSpacePoint;
+use Neos\ContentRepository\Core\Factory\ContentRepositoryId;
```

Comment:
that seems a bit illegal that the shared kernel accesses `Factory` 🤯

Reply:
Little `F6` action https://github.com/neos/neos-development-collection/pull/4891